ts/utils/util.py

```diff
@@ -60,7 +60,7 @@ def map_class_to_label(probs, mapping=None, lbl_classes=None):
     Given a list of classes & probabilities, return a dictionary of
     { friendly class name -> probability }
     """
-    if not (isinstance(probs, list) and isinstance(probs, list)):
+    if not (isinstance(probs, list) and isinstance(lbl_classes, list)):
```

Comment:
I don't think this is correct behavior, 

`lbl_classes` could be `None` and that's OK so the fix should be instead

```suggestion
    if not (isinstance(probs, list):
```